halyard-cli/src/main/java/com/netflix/spinnaker/halyard/cli/command/v1/config/providers/account/AccountCommandProperties.java

```diff
@@ -21,4 +21,8 @@
   public static final String REQUIRED_GROUP_MEMBERSHIP_DESCRIPTION = "A user must be a member of at least one specified group in order to make changes to this account's cloud resources.";
 
   public static final String PROVIDER_VERSION_DESCRIPTION = "Some providers support multiple versions/release tracks. This allows you to pick the version of the provider (not the resources it manages) to run within Spinnaker.";
+
+  public static final String READ_PERMISSION_DESCRIPTION = "A user must have matching roles of all of these read permissions in order to view this account's cloud resources.";
```

Comment:
A user must have _at least one`_ of these roles in order to be granted the `READ` permission. Same with the write description